### PR TITLE
model(engine): support ext field in MasterMetaKVData

### PR DESCRIPTION
**Pull request by liuzix**
### What problem does this PR solve?

Issue Number: ref #6270

### What is changed and how it works?
- Added an `ext` field in `MasterMetaKVData` for better extensibility.

#### Tests 

 - Unit test
 - Integration test

##### Will it cause performance regression or break compatibility?
No.

##### Do you need to update user documentation, design documentation or monitoring documentation?
No.

### Release note 

```release-note
None.
```

### CODE REVIEW

**Review by amyangfei — APPROVED** [review]

File: engine/framework/model/master.go [hunk 1]
```diff
@@ -40,6 +43,56 @@ const (
 	// extend the status code here
 )
 
+// MasterMetaExt stores some attributes of job masters that do not need
```
Comment (maxshuang) [inline engine/framework/model/master.go:46]: Does sqlite mock client support this json?

**Comment by maxshuang** [comment]
Rest LGTM

File: engine/framework/model/master.go (hunk 1 above)
Comment (liuzix) [reply engine/framework/model/master.go:46]: It seems working for now. We may encounter problem if we were to use advanced JSON functions, but they do not seem needed for now.

**Review by maxshuang — APPROVED** [review]

**Pull request merged**